libcontainer/container_linux.go

```diff
@@ -775,6 +775,27 @@ func (c *linuxContainer) updateState(process parentProcess) error {
 	return json.NewEncoder(f).Encode(state)
 }
 
+func (c *linuxContainer) checkFreezer() (Status, error) {
+	path := c.cgroupManager.GetPaths()["freezer"]
+	if _, err := os.Stat(path); err != nil {
+		if os.IsNotExist(err) {
+			// Freezer subsystem is not available,Application is neither Checkpointed nor Destroyed
+			// return the status as Running
```

Comment:
The commend seems a bit odd to me, isn't `Checkpointed` already checked before this function is called?